docs/sphinx/applications/python/krylov.ipynb

```diff
@@ -7,39 +7,45 @@
-    "$$ \\ket{\\psi_{\\alpha}} \\equiv \\ket{\\psi_I^{(n)}} = \\hat{U}_n\\ket{\\Phi_I} $$\n",
+    "From this, a non-orthogonal Krylov Space $\\mathcal{K} = \\{\\psi_{0} \\cdots \\psi_{N}\\}$ is constructed by applying a family of $s$ unitary operators on each of the $d$ reference states resulting in $d*s = N$ elements in the Krylov space where \n",
+    "$$ \\ket{\\psi_{\\alpha}} \\equiv \\ket{\\psi_I^{(n)}} = \\hat{U}_n\\ket{\\Phi_I}, $$\n",
     "\n",
     "Therefore, the general quantum state that we originally set out to describe is\n",
     "\n",
-    "$$ \\ket{\\Psi} = \\sum_{\\alpha} c_{\\alpha}\\ket{\\psi_{\\alpha}} = \\sum_{I=0}^d \\sum_{n=0}^s c_I^{(n)}\\hat{U}_n\\ket{\\Phi_I}  $$\n",
+    "$$ \\ket{\\Psi} = \\sum_{\\alpha} c_{\\alpha}\\ket{\\psi_{\\alpha}} = \\sum_{I=0}^d \\sum_{n=0}^s c_I^{(n)}\\hat{U}_n\\ket{\\Phi_I}.  $$\n",
     "\n",
     "The energy of this state can be obtained by solving the generalized eigenvalue problem\n",
-    "$$ \\boldsymbol{Hc}=\\boldsymbol{Sc}E $$\n",
+    "$$ \\boldsymbol{Hc}=\\boldsymbol{Sc}E, $$\n",
     "\n",
-    "Where the elements of the overlap and Hamiltonian matrix are\n",
+    "where the elements of the overlap  are\n",
     "\n",
     "$$S_{\\alpha \\beta} = \\braket{\\psi_{\\alpha}|\\psi_{\\beta}} =   \\braket{\\Phi_I|\\hat{U}_m^{\\dagger}\\hat{U}_n|\\Phi_J}$$ \n",
     "\n",
-    "$$H_{\\alpha \\beta} = \\braket{\\psi_{\\alpha}|\\hat{H}|\\psi_{\\beta}} =   \\braket{\\Phi_I|\\hat{U}_m^{\\dagger}\\hat{H}\\hat{U}_n|\\Phi_J}$$\n",
+    "and Hamiltonian matrix is\n",
     "\n",
-    "These matrix elements are computed using a quantum computer and the Hadamard test with a circuit shown below for the case of the overlap matrix elements (The Hamiltonian matrix elements circuit would include controlled application of the Hamiltonian in the circuit).  Once the matrices are constructed, the diagonalization is performed classically to produce an estimate for the ground state in question.\n",
+    "$$H_{\\alpha \\beta} = \\braket{\\psi_{\\alpha}|\\hat{H}|\\psi_{\\beta}} =   \\braket{\\Phi_I|\\hat{U}_m^{\\dagger}\\hat{H}\\hat{U}_n|\\Phi_J}.$$\n",
+    "\n",
+    "The matrix elements for $S$ are computed with the Hadamard test with a circuit shown below for the case of the overlap matrix elements. \n",
     "\n",
     "![Htest](./images/krylovcircuit.png)\n",
     "\n",
-    "The $2\\sigma_+$ term refers to measurement of the expectation value of this circuit with the $X+iY$ operator. \n"
+    "The $2\\sigma_+$ term refers to measurement of the expectation value of this circuit with the $X+iY$ operator. \n",
+    "\n",
+    "The Hamiltonian matrix elements are computed with a circuit that includes controlled application of the Hamiltonian.  Once the $H$ and $S$ matrices are constructed, the diagonalization is performed classically to produce an estimate for the ground state in question.\n",
```

Comment:
```suggestion
    "The Hamiltonian matrix elements are computed with a circuit that includes controlled application of the Hamiltonian. Once the $H$ and $S$ matrices are constructed, the diagonalization is performed classically to produce an estimate for the ground state in question.\n",
```

Reply:
The two spaces after a period is kind of IBM Selectric style these days, but it looks like this document is using it everywhere, so probably stick with it here.